### PR TITLE
Add the fingerprinter for Kibana

### PR DESCRIPTION
**Pull request by C4o**
Hi @tooryx,

This PR is about #134.

I fingerprinted the lastest 86 vesions for Kibana 8.x and 7.x.

### CODE REVIEW

File: google/fingerprinters/web/scripts/updater/community/kibana/update.sh [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+source ../../common.sh
+
+SCRIPT_PATH="$(cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P)"
+# Root path to the web fingerprinter plugin.
+PROJECT_ROOT="$(cd -- "${SCRIPT_PATH}/../../../.." >/dev/null 2>&1 ; pwd -P)"
+# Path to the configurations for starting a live instance of Elasticsearch and Kibana.
+ES_APP_PATH="${SCRIPT_PATH}/app"
+# Path to the temporary data holder.
+TMP_DATA="/root/es_fingerprints"
```
Comment (leonardo-doyensec) [inline google/fingerprinters/web/scripts/updater/community/kibana/update.sh:27]: Please change this folder to `/tmp/es_fingerprints`.

**Review by leonardo-doyensec — COMMENTED** [review]
Hello @C4o.
Thank you for your contribution. I'm noticing that there are newer versions of Kibana. Please add those to the `version.txt` file and commit the new `.binproto` file.

Moreover you can find an issue to address down below.

Feel free to reach out
~ Leonardo (Doyensec)

**Comment by C4o** [comment]
Hi @leonardo-doyensec, I've updated.

**Comment by leonardo-doyensec** [comment]
Hi @C4o, thank you for your changes. I'm noticing that the version file is still the same. Can you please update it with all the new version of Kibana. After that, please re-run the script in order to commit the `.binproto` file with all the new fingerprints.

**Comment by C4o** [comment]
Hi @leonardo-doyensec, I've updated .binproto file.

**Comment by leonardo-doyensec** [comment]
**LGTM**
@maoning we can merge this

**Review by tooryx — APPROVED** [review]

**Comment by tooryx** [comment]
Hi @C4o,

This PR is being reviewed internally. It should be merged in a few days.

~tooryx

**Pull request merged**